src/lesson4Array/Array.java

```diff
@@ -0,0 +1,65 @@
+package lesson4Array;
+
+import java.util.Scanner;
+
+public class Array {
+    private int length;
+    private int[] dataTable;
+    private static Scanner scanner = new Scanner(System.in);
+
+    public Array(int length) {
+        this.length = length;
+        this.dataTable = new int[length];
+    }
+
+    public int setData() {
+        for (int i = 0; i < dataTable.length; i++) {
+
+            dataTable[i] = scanner.nextInt();
+        }
+
+        return 0;
+    }
+
+    public int calculateSum() {
+
+        if (dataTable == null) {
+            return 0;
```

Comment:
smart move. However, I would call the setData() methods instead of return 0. Therefore, if the dataTable is null then the user will be asked to enter some values.